pytest_mypy_plugins/configs.py

```diff
@@ -54,6 +55,19 @@ def join_toml_configs(
     with mypy_config_file_path.open("w") as f:
         # We don't want the whole config file, because it can contain
         # other sections like `[tool.isort]`, we only need `[tool.mypy]` part.
-        f.write(f"{_TOML_TABLE_NAME}\n")
-        f.write(dedent(toml_config["tool"]["mypy"].as_string()))  # type: ignore[index]
+        tool_mypy = toml_config["tool"]["mypy"]
+
+        # construct toml output
+        min_toml = tomlkit.document()
+        min_tool = tomlkit.table(is_super_table=True)
+        min_toml.append("tool", min_tool)
+        min_tool.append("mypy", tool_mypy)
+
+        # strip leading comments
+        for i, item in enumerate(min_toml.body):
+            if not isinstance(item, (Comment, Whitespace)):
+                break
+        min_toml.body[:] = min_toml.body[i:]
```

Comment:
```suggestion
        min_toml.body = min_toml.body[i:].copy()
```

This seems to be the same thing, but easier to read?

Reply:
`tomlkit.item.Container.body` is read-only:

https://github.com/python-poetry/tomlkit/blob/911cccd630965ff423316e25b4685ecf7df0ec0a/tomlkit/container.py#L42-L44

So this is the only way to modify in place.

Reply:
Nevermind.
My local installation was broken. This code path was not required and never used.

I removed it.